x/gamm/pool-models/internal/test_helpers/test_helpers.go

```diff
@@ -0,0 +1,63 @@
+package test_helpers
+
+import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/store/rootmulti"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/libs/log"
+	tmtypes "github.com/tendermint/tendermint/proto/tendermint/types"
+	dbm "github.com/tendermint/tm-db"
+
+	"github.com/osmosis-labs/osmosis/v7/osmoutils"
+	"github.com/osmosis-labs/osmosis/v7/x/gamm/types"
+)
+
+// CfmmCommonTestSuite is the common test suite struct of Constant Function Market Maker,
+// that pool-models can inherit from.
+type CfmmCommonTestSuite struct {
+	suite.Suite
```

Comment:
Why aren't we just using apptesting here?

Reply:
We could also delete the CreateTestContext if we used apptesting

Reply:
It's because we have import cycle problems when we do that since the apptesting package imports gamm module.